src/Parser/Header/SecChUaPlatformDevice.php

```
@@ -0,0 +1,52 @@
+use function mb_strtolower;
+use function mb_trim;
+
+final class SecChUaPlatformDevice implements DeviceCodeInterface
+{
+    /**
+     * @throws void
+     *
+     * @phpcs:disable SlevomatCodingStandard.Functions.UnusedParameter.UnusedParameter
+     */
+    #[Override]
+    public function hasDeviceCode(string $value): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$value'. (undefined)

(UnusedFormalParameter)